src/Microsoft.DotNet.ProjectJsonMigration/ConstantPackageNames.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectModel;
+using System.IO;
+using Newtonsoft.Json.Linq;
+using System.Text;
+
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+    public class ConstantPackageNames
+    {
+        public const string CSdkPackageName = "Microsoft.DotNet.Core.Sdk";
```

Comment:
ty, I have been cognizant of this, but haven't seen us take the new one in Cli, when we do tests will fail until we change this